File: src/java/checkout.java
String data = "{\n  \"merchant_key\":\"MERCHANT KEY\",\n  \"public_key\":\"PUBLIC KEY\",\n  \"callback_url\":\"https://www.mydomain.com/success.html\",\n  \"return_url\":\"https//www.mydomain.com/return.html\",\n  \"tx_ref\":\"REF_usergenerated\",\n  \"amount\":\"10000\",\n  \"email\":\"mail@example.com\",\n  \"first_name\":\"Finn\",\n  \"last_name\":\"Marshal\",\n  \"title\":\"RPI\",\n  \"description\":\"Payment For RPI\",\n  \"quantity\":\"10\",\n  \"currency\":\"87\"\n}";

byte[] out = data.getBytes(StandardCharsets.UTF_8);
OutputStream stream = http.getOutputStream();

File: src/javascript/checkout.js
var url = "https://sandbox.helapass.com/api/mer_transfer";

var xhr = new XMLHttpRequest();
xhr.open("POST", url);

xhr.setRequestHeader("Content-Type", "application/json");

xhr.onreadystatechange = function () {
   if (xhr.readyState === 4) {
      console.log(xhr.status);
      console.log(xhr.responseText);
   }};

var data = `{
  "merchant_key":"MERCHANT KEY",
  "public_key":"PUBLIC KEY",
  "callback_url":"https://www.mydomain.com/success.html",
  "return_url":"https//www.mydomain.com/return.html",
  "tx_ref":"REF_usergenerated",
  "amount":"10000",
  "email":"mail@example.com",
  "first_name":"Finn",
  "last_name":"Marshal",
  "title":"RPI",
  "description":"Payment For RPI",
  "quantity":"10",
  "currency":"174"
}`;

xhr.send(data);
